src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpRedirectFeaturesAnalyzers.cs

```diff
@@ -143,14 +145,16 @@ await TestServices.Workspace.WaitForAllAsyncOperationsAsync(
                 FeatureAttribute.ErrorList
              ],
              cancellationToken);
-        return await TestServices.ErrorList.GetErrorsAsync(cancellationToken);
+        return await TestServices.ErrorList.GetErrorsAsync(ErrorSource.Other, __VSERRORCATEGORY.EC_WARNING, cancellationToken);
     }
 
     private async Task WaitForCodeActionListToPopulateAsync(CancellationToken cancellationToken)
     {
         await TestServices.Editor.ActivateAsync(cancellationToken);
         await TestServices.Editor.PlaceCaretAsync("void M()", charsOffset: -1, cancellationToken);
 
+        await Task.Delay(TimeSpan.FromSeconds(1));
```

Comment:
😬 Why this?

Reply:
Note that `InvokeCodeActionListAsync` invokes `WaitForLightBulbAsync`, so this delay will not improve stability.